src/test/functional/features/court-details.feature

```diff
@@ -234,3 +234,15 @@ Feature: Court Name Know - Court Details
     Examples:
       | no_access_scheme_court |
       | Chancery Division      |
+
+  Scenario Outline: Not-in-person Court - Get update on application
+    And I have entered "<centre_that_should_allow_update_on_application>" as search criteria
+    When I have selected to search for that court or tribunal name or address
+    Given results are returned
+    When I select a court or tribunal link
+    And that location is a 'not-in-person' court or tribunal
+    Then the get an update on my application section is displayed
```

Comment:
I wonder we should add a further step here to test that we can click on the link of an entry within the application update section? Let me know your thought

Reply:
We could add something like '**And** there is a link to the "Online money claims" website at url "foo"` and then the step definition checks for an anchor tag with that URL.

Reply:
I've asked Ashwini to comment on this. We could definitely add a step that checks for a specific link and URL but I'm slightly concerned that, in time, if this section is editable in the admin portal, someone changing or removing a link for this court would cause our test to fail.

Reply:
I have had a chat with Ashwini and we agreed that checking the section is displayed is sufficient here. QA will also be able to review the functional test during their review.

Reply:
ok that's fine. We will just leave it like that :)